gradle.properties

```diff
@@ -12,7 +12,6 @@ protobufJavaVersion=3.9.1
 jknackHandlebarsVersion=4.0.6
 nettyVersion=4.1.71.Final
 picocliVersion=4.0.1
-nettyTcnativeVersion=2.0.46.Final
 bouncycastleVersion=1.69
 wso2CommonsPoolVersion=1.5.6.wso2v1
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/598/commits/d192cd4e00a73effa84122cc36cd949d75457216